### ISSUE TITLE
Request: Add interlude tag

### ISSUE THREAD

**Issue by GenaBitu** [DNepovim/zabekame#11]

**Comment by DNepovim** [comment]
For what? It's only line of chords. Right?

**Comment by GenaBitu** [comment]
Yes, but it would be nicer if there were a gray I: like there is for chorus and verse. Also could automatically insert the chord on a line with just <i>, just like <ch> does.
